locust/contrib/fasthttp.py

```diff
@@ -92,7 +92,6 @@ def __init__(self, base_url):
                 max_retries=1,
                 cookiejar=self.cookiejar,
                 ssl_context_factory=create_default_context,
-                ssl_options={"server_hostname": parsed_url.hostname},
```

Comment:
removing this will fix the current case, but will break activities that try to use SNI to select from multiple certificates (e.g., section and enrollment requests, or anything else using the API gateway). Do we know why `parsed_url.hostname` contains the wrong hostname? Are we following a redirect or something else that should be modifying the value we send for `server_hostname` ?

Reply:
No idea, I'm gonna try to debug and find out